docs/docs/.vuepress/components/Footer.vue

```diff
@@ -5,7 +5,7 @@
     </h2>
     <p>
       Please
-      <a href="/support/" class>reach out</a> if this doc didn't answer your question. We're happy to help!
+      <a href="https://pipedream.com/support/" class>reach out</a> if this doc didn't answer your question. We're happy to help!
```

Comment:
I'm replacing `/support` links with https://pipedream.com/support everywhere in the docs.